### DISCUSSION TITLE
Reusing views from one coordinator in another one.

### DISCUSSION THREAD

**Discussion by sparafinski** [johnpatrickmorgan/FlowStacks#66] [Q&A]
I am trying to reuse view created in one coordinator and place it in another one. 
The problem I encounter is that the called view is showed as it should, but all actions called on the navigator (`FlowNavigator<>`) inside subordinate views cause the application to crash due to:

> No ObservableObject of type FlowNavigator<FirstTabViews> found. A View.environmentObject(_:) for FlowNavigator<FirstTabViews> may be missing as an ancestor of this view.

What's interesting is that the "first" call on the navigator seems to work as expected - the same cell that is inside the list I'm trying to display opens the dependent view without a problem - only nested navigators seem to have a problem. 

I noticed that setting the key `embedInNavigationView` to `true` fixes this error a bit, however the subordinate views are pushed directly to where the coordinator view is located and not as initially in "proper" but not quite working navigationView. 

For any ideas and help I will be grateful 💪 

There you can find demo project with the issue: 
https://github.com/sparafinski/FlowStacksDemo

And there is a short video presenting the issue: 

https://github.com/johnpatrickmorgan/FlowStacks/assets/40520109/9d57d810-8259-4eac-a1c5-a345a29ac4a0

**Comment by kunal-jar** [comment]
@sparafinski Hello were you able to find a solution for the same? cc @johnpatrickmorgan

**Comment by johnpatrickmorgan** [comment]
Hi @kunal-jar - this issue was first raised a while ago and the library has changed a lot since then. Please could you open a new issue to describe the problem you're having? Thanks!
